src/sentry/api/endpoints/group_participants.py

```diff
@@ -1,15 +1,18 @@
+from __future__ import annotations
+
 from rest_framework.request import Request
 from rest_framework.response import Response
 
 from sentry.api.base import region_silo_endpoint
 from sentry.api.bases import GroupEndpoint
-from sentry.api.serializers import serialize
-from sentry.models import GroupSubscriptionManager
+from sentry.services.hybrid_cloud.user import user_service
 
 
 @region_silo_endpoint
 class GroupParticipantsEndpoint(GroupEndpoint):
```

Comment:
GroupEndpoint has logic to check for the matching and existence of the group + organization with `organization_slug`